database/dataGenerator.js

```diff
@@ -28,10 +29,19 @@ var getRandomRecords = function() {
       decorScore,
       serviceScore
     }
+    oneRestaurant.push(name, type, price, location, foodScore, decorScore, serviceScore);
+    allRestaurants.push(oneRestaurant);
```

Comment:
are you creating an array of arrays? IIRC mysql takes in an array of objects so you might want to double check ^^

Reply:
I successfully seeded the database last night using my original array of arrays, so maybe it can take either? Thanks Eugenia!